task-list/app/views/tasks/index.html.erb

```diff
@@ -1 +1,10 @@
-<h1>I WORK</h1>
+<div class="content">
+  <h1>This took a lot of work. I promise.</h1>
+  <ul>
+    <% @tasks.each do |task| %>
+    <li>
+    <a href="/tasks/<%= task.id %>"><%= task.name %>: <%= task.description %></a>
+    <% end %>
```

Comment:
I would highly recommend that your ruby code do/end matches starting and closing tags. Here, you open a loop, open an `li` tag, then close the loop, then close the `li` tag. This should close the `li` before you close the loop.